### PR TITLE
Send emails to directory during development

### PR DESCRIPTION
**Pull request by TravCav**
In response to Issue #235. I've changed Web.config and
Web.Release.config to have SmtpMessageSender.SendMessage drop emails
into C:\Users\[USER]\AppData\Roaming while developing and send via smtp
server when in release mode.

### CODE REVIEW

File: crisischeckin/Services/MessageCoordinator.cs [hunk 1]
```diff
@@ -21,9 +21,10 @@ public void SendMessage(Message message, List<MessageRecipient> recipients)
                 {
                     messageSender.SendMessage(message, recipients);
                 }
-                catch (Exception)
+                catch (Exception e)
                 {
                     //TODO: Add logging implementation--awaiting decision from Bill
+                    ////throw e;
```
Comment (BillWagner) [inline crisischeckin/Services/MessageCoordinator.cs]: We already have elmah as part of the project. Can you update this pull request to include the logging here? Thanks.

**Comment by TravCav** [comment]
Removed the try/catch from MessageCoordinator.SendMessage.  I don't have
experience with elmah. Did some research and I'm assuming elmah takes care of everything?  The try/catch and //TODO were already there when I got there. Was there a prior conversation around that method with someone?

**Comment by TravCav** [comment]
@BillWagner @tonysurma Can I get some advice on what this needs?

**Comment by tditiecher** [comment]
While reading through the code for this project (as I would like to participate), I saw something unexpected: in the SmtpMessageSender the SmtpClient is created using new SmtpClient() instead of using the injected smtpClientFactory. I expected the configuration of the SmtpClient (PickupDirectory) in debug mode to be in  NinjectWebCommon.RegisterServices.

**Comment by BillWagner** [comment]
@tditiecher That's a great comment. I'll approve this PR, which handles some of the other issues, and I'd like to have you open a separate issue on that, and please take care of it.

**Pull request merged**